apis/fluent/atrium-api-fluent/src/jvmTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/BigDecimalExpectationsSpec.kt

```diff
@@ -14,7 +14,7 @@ class BigDecimalExpectationsSpec : Spek({
         @Suppress("DEPRECATION") fun1<BigDecimal?, BigDecimal?>(Expect<BigDecimal?>::toEqual).withNullableSuffix(),
         fun1<BigDecimal?, Nothing?>(Expect<BigDecimal?>::toEqual).withNullableSuffix(),
         Expect<Any>::toEqual,
-        @Suppress("DEPRECATION") (Expect<BigDecimal>::notToEqual.name to  Expect<BigDecimal>::notToEqual),
+        @Suppress("DEPRECATION") (Expect<BigDecimal>::notToEqual.name to { notToEqual(it) }),
```

Comment:
This change was needed to call the deprecated `Expect<BigDecimal?>.notToEqual(BigDecimal?)`.
I do not know if there's a way to disambiguate `Expect<BigDecimal>::notToEqual` between `notToEqual(BigDecimal?) `and `notToEqual(Nothing?)`

I think having a lambda here should be good enough.